src/Caching/SimpleStringCache.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Caching;
+
+/**
+ * This cache caches string values with string keys. It is not PSR-6-compliant.
+ *
+ * Usage:
+ *
+ * ```php
+ * $cache = new SimpleStringCache();
+ * $cache->set($key, $value);
+ * …
+ * if ($cache->has($key) {
+ *     $cachedValue = $cache->get($value);
+ * }
+ * ```
+ */
+class SimpleStringCache
+{
+    /**
+     * @var array<string, string>
+     */
+    private $values = [];
+
+    /**
+     * Checks whether there is an entry stored for the given key.
+     *
+     * @param string $key the key to check; must not be empty
+     *
+     * @throws \InvalidArgumentException
+     */
```

Comment:
Should there be `@return bool` or is this superfluous as it does not provide any additional information?

Reply:
I think it's superfluous.

Reply:
OK, I kind-of thought so too.  (Also, we have recently removed insistence on superfluous DocBlock annotations from the sniffs.  We could perhaps even insist on not having superfluous DocBlock annotations if there is a sniff for it.)